joy2twist/include/joy2twist/joy2twist_node.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef JOY2TWIST_JOY2TWIST_NODE_HPP
+#define JOY2TWIST_JOY2TWIST_NODE_HPP
```

Comment:
according to our code style there should be trailing `_`

Reply:
:heavy_check_mark: